src/components/pages/order/adminPanel/PanelAdminTabs.js

```diff
@@ -7,9 +7,11 @@ import BlocTabs from "./BlocTabs";
 export default function PanelAdminTabs() {
   const { isModeAdmin } = useContext(GlobalContext);
 
+  const bloc_tab_container = isModeAdmin ? "showAdminPanel" : "closeAdminPanel";
```

Comment:
```suggestion
  const blocTabContainer = isModeAdmin ? "showAdminPanel" : "closeAdminPanel";
```

C'est une variable classique, donc pourquoi vouloir changer la notation en snake case ? Pas besoin ici. Le Camel case suffit largement. Il faut éviter autant que possible de multiplier différents types de notation dans un même projet.

Reply:
ok